_component_design/buttons-drop-down.md

```diff
@@ -11,7 +11,7 @@ resource: true
 last-modified: 2018-09-06
 helix-ui-css: true
 helix-ui-javascript: na
-pagelink: https://rackerlabs.github.io/helix-ui/components/menus/#grouped-menu
+pagelink: https://rackerlabs.github.io/helix-ui/components/menu/#grouped-menu
```

Comment:
Do we need the anchor link (`#grouped-menu`)? A dropdown button is just a menu. It can be grouped or simple.

Reply:
I think it was maybe at one point different? I don't remember, but happy to just drop the anchor link if it makes more sense. I haven't gotten around to checking these in a while.